src/components/button/test-button.js

```diff
@@ -30,6 +30,11 @@ describe('Button', function() {
     };
   });
 
+  it('matches the snapshot', function() {
```

Comment:
While this is a good first step for checking the output, the updates to the tests below removed the checks that the correct classes are applied with the type changes. Any time there is a conditional in the component we should have a test to make sure that it's actually generating what we expect.

Reply:
Take a look at the resulting snapshots for your updates....the output is identical for all test cases...